src/test/java/picard/util/IntervalListToolsTest.java

```diff
@@ -115,6 +116,25 @@ public void testAllActions(final IntervalListTools.Action action) throws IOExcep
         Assert.assertEquals(runPicardCommandLine(args), 0);
     }
 
+    // test that all actions work for standard input, but not test output at all.
+    @Test(dataProvider = "ActionsTest")
```

Comment:
Since this test is essentially identical to the one above, it would be better to modify the @dataprovider so that it creates objects of the form `{IntervalListTools.Action, File}`, and provide the input file as a second argument to `testAllActions`